packages/cactus-plugin-ledger-connector-besu/src/main/typescript/plugin-ledger-connector-besu.ts

```diff
@@ -389,8 +399,55 @@ export class PluginLedgerConnectorBesu
     const method: ContractSendMethod = methodRef(...req.params);
 
     if (req.invocationType === EthContractInvocationType.Call) {
-      const callOutput = await (method as any).call();
-      const success = true;
+      let callOutput;
```

Comment:
@travis-payne Please open a follow-up issue to come up with a way of simplyfing the code here, the length of the method and the number of if conditions starts to smell here. No need to re-work anything in the context of this current PR because this one really needs to move along now, but I want to make sure we track it that this has to be improved.

@TonyRowntree is already working on some other PRs that make the keychain optional by splitting the endpoints to smaller ones so we have to also cross-coordinate with him/his PRs. There will be conflicts, but we have to get this code complexity under control before the 1.0 release because then all  breaking changes will have to wait until Cactus 2.0.

cc: @m-courtin